src/components/ChartContainer.js

```diff
@@ -295,44 +275,73 @@ const ChartContainer = forwardRef(
     }));
 
     return (
-      <div
-        ref={container}
-        className={"orgchart-container " + containerClass}
-        onWheel={zoom ? zoomHandler : undefined}
-        onMouseUp={pan && panning ? panEndHandler : undefined}
+      <TransformWrapper
+        initialScale={1}
+        ref={transformComponentRef}
+        disabled={!(!!zoom)}
+        minScale={zoom && zoom.minZoom}
+        maxScale={zoom && zoom.maxZoom}
+        doubleClick={{
+          disabled: true
+        }}
+        wheel={{
+          disabled: true
+        }}
+        panning={{
+          disabled: true
+        }}
       >
-        <div
-          ref={chart}
-          className={"orgchart " + chartClass}
-          style={{ transform: transform, cursor: cursor }}
-          onClick={clickChartHandler}
-          onMouseDown={pan ? panStartHandler : undefined}
-          onMouseMove={pan && panning ? panHandler : undefined}
-        >
-          <ul>
-            <ChartNode
-              datasource={attachRel(ds, "00")}
-              NodeTemplate={NodeTemplate}
-              draggable={draggable}
-              collapsible={collapsible}
-              multipleSelect={multipleSelect}
-              changeHierarchy={changeHierarchy}
-              onClickNode={onClickNode}
-            />
-          </ul>
-        </div>
-        <a
-          className="oc-download-btn hidden"
-          ref={downloadButton}
-          href={dataURL}
-          download={download}
+        <TransformComponent
+          wrapperClass="transform-wrapper"
+          contentClass="transform-content"
+          wrapperStyle={{
+            minWidth: "100%",
+            maxHeight: "calc(100vh - 50px)",
```

Comment:
Curious what the 50px is?

Reply:
It's a mistake from a prior implementation :). I will be removing it shortly

Reply:
😄